### PR TITLE
changed LM metric reporting calculation

### PR DESCRIPTION
**Pull request by shreydesai**
Summary: Changes LM metric reporting calculation to aggregate losses and # words/batch as opposed to appending logits (potentially very large matrices) at every time step.

Differential Revision: D16031948

### CODE REVIEW

**Comment by Titousensei** [comment]
Abandoned
